internal/upgrader/upgrader_test.go

```diff
@@ -181,55 +181,84 @@ var _ = Describe("Upgrade", func() {
 	})
 
 	When("running with --check-up-to-date", func() {
-		It("should print out service GUIDs and not attempt to upgrade", func() {
-			result := captureStdout(func() {
-				l := logger.New(100 * time.Millisecond)
-				defer l.Cleanup()
-				err := upgrader.Upgrade(fakeCFClient, l, upgrader.UpgradeConfig{
-					BrokerName:       fakeBrokerName,
-					ParallelUpgrades: 5,
-					CheckUpToDate:    true,
-				})
-				Expect(err).To(MatchError("check up-to-date failed: found 3 instances which are not up-to-date"))
-			})
+		Context("there are instances not up to date", func() {
+			var notUpToDateInstance, withPlanDeactivated ccapi.ServiceInstance
+			BeforeEach(func() {
+				fakeCFClient.GetServicePlansReturns([]ccapi.ServicePlan{
+					{
+						GUID:                   "fake-plan-1-guid",
+						Available:              true,
+						Name:                   "fake-plan-1-name",
+						MaintenanceInfoVersion: "1.5.7",
+					},
+					{
+						GUID:                   "fake-plan-2-guid",
+						Available:              true,
+						Name:                   "fake-plan-2-name",
+						MaintenanceInfoVersion: "1.5.7",
+					},
+				}, nil)
 
-			By("getting the service plans")
-			Expect(fakeCFClient.GetServicePlansCallCount()).To(Equal(1))
-			Expect(fakeCFClient.GetServicePlansArgsForCall(0)).To(Equal(fakeBrokerName))
+				fakeInstance1 = ccapi.ServiceInstance{
```

Comment:
issue(blocking): This is the very definition of upgradeable, the example is misleading. Instance is at 1.3.0 and plan at 1.5.7, `UpgradeAvailable` should be `true`.